aviatrix/data_source_aviatrix_firewall_instance_images.go

```diff
@@ -61,8 +68,12 @@ func dataSourceAviatrixFirewallInstanceImagesRead(d *schema.ResourceData, meta i
 	for _, image := range *firewallInstanceImages {
 		fI := make(map[string]interface{})
 		fI["firewall_image"] = image.Image
-		fI["firewall_image_version"] = image.Version
-		fI["firewall_size"] = image.Size
+		listVersion := image.Version
+		sort.Sort(sort.Reverse(byVersion(listVersion)))
```

Comment:
You should be able to use sort.Slice instead of sort.Sort. That way, you only need to define 2 new Less functions and you don't have to define new types.

Reply:
fix:
sort.Slice(versionList, func(i, j int) bool {
 			return sortVersion(versionList, i, j)
 		})
 fI["firewall_image_version"] = reverseArray(versionList)
 sizeList := image.Size
 sort.Slice(sizeList, func(i, j int) bool {
 			return sortSize(sizeList, i, j)
 		})